### DISCUSSION TITLE
How to fill data from database or API into Form build by OrchardCMS

### DISCUSSION THREAD

**Discussion by ThangNguyen1316** [OrchardCMS/OrchardCore#10164] [Q&A]
I want to set default value from database or API into a input of form generated by OrchardCMS, something like edit product. How can I do that ?

**Comment by hishamco** [comment]
Is it a custom database or API?

**Reply by Spiderman99999999** [reply]
It’s from database

**Comment by ns8482e** [comment]
Not sure if Handler can be helpful here, but you can try implementing your own `ContentPartHandler<FormPart>` for  `FormPart`
